vault/vault/config/vault-config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "backend": {
+    "file": {
+      "path": "vault/data"
+    }
+  },
+  "listener": {
+    "tcp":{
+      "address": "0.0.0.0:8200",
+      "tls_disable": 1
+    }
+  },
+  "storage": {
+    "postgresql": {
+      "connection_url": "postgres://vault:vault@database:5432/vault?sslmode=disable",
```

Comment:
why it's hardcoded username/password ?

Reply:
https://www.vaultproject.io/docs/configuration/storage/postgresql#postgresql-parameters